src/gui/masterkey/KeyFileEditWidget.ui

```diff
@@ -63,6 +63,13 @@
      </property>
     </spacer>
    </item>
+   <item row="1" column="0">
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>Do not use a file that may change! Your database would be lost forever.</string>
+     </property>
+    </widget>
+   </item>
```

Comment:
@droidmonkey like this?

Reply:
The label text feels a bit strange to me, but Im not a native speaker, and I don't have a better idea. Feel free to suggest something